jmix-quartz/quartz/src/main/java/io/jmix/quartz/service/QuartzService.java

```diff
@@ -114,6 +135,15 @@ from the startTime to the current moment after trigger is recreated (all trigger
         return result;
     }
 
+    public String getDisplayedClassName(JobModel jobModel, JobDetail jobDetail) {
```

Comment:
Why do you need JobModel here?
jobDetail.getJobClass().getName()

Reply:
ok